engine/src/main/scala/cromwell/engine/EngineFilesystems.scala

```diff
@@ -28,10 +29,11 @@ case class EngineFilesystems(actorSystem: ActorSystem) {
   }
   
   private val defaultFileSystem = if (config.as[Boolean]("engine.filesystems.local.enabled")) {
-    Option(DefaultPathBuilder)
+    Option(Future.successful(DefaultPathBuilder))
   } else None
 
-  def pathBuildersForWorkflow(workflowOptions: WorkflowOptions): List[PathBuilder] = {
-    List(gcsPathBuilderFactory map { _.withOptions(workflowOptions)(actorSystem) }, defaultFileSystem).flatten
+  def pathBuildersForWorkflow(workflowOptions: WorkflowOptions)(implicit as: ActorSystem, ec: ExecutionContext): Future[List[PathBuilder]] = {
+    val maybeEventualBuilders = List(gcsPathBuilderFactory map { _.withOptions(workflowOptions) }, defaultFileSystem).flatten
+    Future.sequence(maybeEventualBuilders)
```

Comment:
Is there an equivalent of `Future` in cats that conceptually wraps `Validated` instead of `Try`?